app/models/spree/gateway/pay_pal_express.rb

```diff
@@ -101,7 +91,7 @@ def express_checkout_url(pp_response, extra_params={})
         "https://www.#{server_domain}paypal.com/cgi-bin/webscr?" +
           "cmd=_express-checkout&force_sa=true&"
       end +
-      encode_www_form(params)
+      URI.encode_www_form(params)
```

Comment:
`URI` is automatically added to the namespace when running Rails, but it's better to require it and call it explicitly here, otherwise specs will fail:

```
Failure/Error: encode_www_form(params)
     
     NoMethodError:
       undefined method `encode_www_form' for #<Spree::Gateway::PayPalExpress:0x00007f8247a01f88>
       Did you mean?  encode_with
```